docs/api/io/flow_control.md

```diff
@@ -2,51 +2,66 @@
 
 We can use Blinky to explore flow control and task management in Arm Mbed OS applications. We'll look at automated actions first, then move on to handling user actions.
 
-## Flow control for automated actions
+## Flow control for delayed actions
 
-If we want to automatically blink an LED, we have three main techniques:
+If we want to automatically blink an LED, we have four main techniques:
 
-1. [Wait](#wait)
-1. [Ticker](#ticker)
+1. [Busy wait](#busy-wait)
+1. [Timer, Ticker, Timeout](#ticker-timeout)
```

Comment:
```suggestion
[Ticker, Timeout](#ticker-timeout)
```
(Timer measures time but does not provide any delay/callback)

Reply:
Was it maybe added because you could use timer as follows:

```
READ CURRENT TIME FROM TIMER AND STORE IN START_TIME
IN A WHILE LOOP:
   IF READ CURRENT TIME FROM TIMER - START_TIME >= DELAY:
       PERFORM ACTION
       READ_CURRENT AND STORE IN START_TIME
```

Reply:
Checking a timer in a while loop isn't recommended is any case, so I'd rather leave it out. I think a timer is mainly for measuring time like benchmarking a procedure, etc. but that's outside the scope of this doc.

Reply:
Oh yes. I just suggested an explanation why it was there in the first place.